### PR TITLE
compile.sh - Automatically choose master branch

### PR DESCRIPTION
**Pull request by SafwatHalaby**
Since the testing branch was [removed](http://forum.mc-server.org/showthread.php?tid=434&pid=23415#pid23415) by @worktycho , `compile.sh` was updated.

### CODE REVIEW

File: compile.sh [hunk 1]
```diff
@@ -84,6 +84,13 @@ if [[ $MISSING_PROGRAMS != "" ]]; then
 	missingDepsExit
 fi
 
+# Bypass Branch choice and choose master. Because it's the only branch right now.
+BRANCH="master"
+
+##Inactive code start
```
Comment (madmaxoft) [inline compile.sh]: I do prefer a space between the comment-sign and the comment text itself, in C++ we have it as a hard requirement, but we should do it in scripts, too

Comment (SafwatHalaby) [inline compile.sh]: Fixed.

**Comment by madmaxoft** [comment]
Still, shouldn't we instead make a new `testing` branch currently pointing to `master`?

**Comment by SafwatHalaby** [comment]
I made this as a quick workaround because currently people will encounter a broken script, which isn't nice.

I'll easily modify it if/when a new testing branch is created.

**Comment by SafwatHalaby** [comment]
Squashed.

**Comment by madmaxoft** [comment]
Fair enough. Merging.

**Pull request merged**